invaitGenerator.py

```diff
@@ -20,18 +20,19 @@
     print()
     shehe = input('Кому адресовано приглашение (ей/ему)?\n')
 elif decision == 'выход':
-	input('Если вновь понадобятся мои услуги, приходите!\n')
+    print('Если вновь понадобятся мои услуги, приходите!\n')
+    exit()
 else:
-    i = 'начать'
-    ii = 'выход'
-    while decision<=i or ii:
-        print( Back.RED )
-        print('Выбрано не правильное действие.\nВыберите начать или выход.')
-        print()
-        print( Back.GREEN )
-        decision2 = input('Приступаем? (начать/выход)\n')    
-        if decision2<=i or ii:
-            break
+    print('Выбранно неправильное действие,\nвыберите (начать/выход).')
+#    i = 'начать'
+#    ii = 'выход'
+#    while decision<=i or ii:
```

Comment:
why you commented out this code? Different task IG-2 supposed to fix it.

Reply:
'"exit from the program added" is the better name for the commit'
well

'Use branch name "IG-1" next time instead of "IG-1_"'
Next time I'm more attentive

'why you commented out this code? Different task IG-2 supposed to fix it.'
I agree, I added too much